modules/social_features/social_profile/modules/social_profile_organization_tag/config/modify/social_profile_organization_tag.taxonomy_permissions.yml

```diff
@@ -0,0 +1,23 @@
+dependencies:
+  module:
+    - social_registration_fields
+    - taxonomy_access_fix
+items:
+  user.role.anonymous:
```

Comment:
Looking at 

```function social_profile_organization_tag_install() {
  // Grant the default permissions for this feature.
  user_role_grant_permissions(
    'contentmanager',
    [
      'delete terms in profile_organization_tag',
      'edit terms in profile_tag',
      'edit profile organization tags',
    ]
  );
  user_role_grant_permissions(
    'sitemanager',
    [
      'delete terms in profile_organization_tag',
      'edit terms in profile_tag',
      'edit profile organization tags',
    ]
  );
  // Set the weight of this module to 1 so it is loaded after
  // the social_profile module.
  module_set_weight('social_profile_organization_tag', 1);
}```

It seems only CM and SM should be able to select terms for this.